### PR TITLE
Added a gallery section in the home page

### PR DESCRIPTION
**Pull request by AtriSukul1508**
I have added a gallery section in the home page and also created a page for gallery section

### CODE REVIEW

File: styles/gallery.css [hunk 1]
```diff
@@ -0,0 +1,135 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+  }
+  body{
+    height: 100vh;
```
Comment (niloysikdar) [inline styles/gallery.css:7]: `height: 100vh` for body 🤔

File: styles/gallery.css [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    overflow-x: hidden;
+    background-image: url('../images/gallery_page_background.jpg');
+    background-size: cover;
+    background-repeat: repeat;
+}
+#gallery_heading{
+  color: #6c63ff;
+}
+.center {
+  width: 100vw;
```
Comment (niloysikdar) [inline styles/gallery.css:17]: since div is a block-level element, shoudn't it take 100% width (100vw) by default?

File: styles/gallery.css [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+  border: none;
+  border-radius: 10px;
+  margin: 10px;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  max-width: 270px;
+  min-height: 300px;
+  width: 100%;
```
Comment (niloysikdar) [inline styles/gallery.css:35]: `max-width` and `width` both ?

File: styles/index.css [hunk 4]
```diff
@@ -1,75 +1,141 @@
...
+    height: 100vh;
+    overflow-x: hidden;
+  }
+  .container-fluid_1{
+    background-image: url('../images/Background.jpg');
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    height: 700px !important;
```
Comment (niloysikdar) [inline styles/index.css:17]: We should always try to avoid using `!important` if it isn't necessary

**Review by niloysikdar — COMMENTED** [review]
Overall it seems fine @AtriSukul1508
Kindly go through the comments and feedbacks, I will test it tomorrow and merge it if everything will work fine.

**Comment by AtriSukul1508** [comment]
@niloysikdar Thank you and I'll solve the issues today

File: js/gallery.js [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+            stateCard[i].style.height="336.797px";
+            stateTitle.style.display="block";
+            stateCard[i].style.transitionDelay= "34s";
+            image.style.transitionDelay= "34s";
+            image.style.display = "block";
+
+    
+        })
+    }
+};
```
[the remark this answers is not in the document]
Comment (AtriSukul1508) [reply js/gallery.js:116]: CSS selector wasn't working properly, that's why I added this event

File: styles/gallery.css (hunk 1 above)
Comment (AtriSukul1508) [reply styles/gallery.css]: I have resolved this issue

**Review by niloysikdar — APPROVED** [review]
Nice work @AtriSukul1508

**Pull request merged**